### ISSUE TITLE
Rights Status for parent works

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-lux#620]
When we create parent-child works, the parent work must receive all required metadata like its children. The children may all have different rights statuses, however, particularly for serial publications published over a long period of time.

The available values for rightststatements.org do not provide an option for an aggregated/variable rights status.

[More info/proposed workarounds](https://docs.google.com/document/d/12HQ5YrLG06ODOEQNJwdwdm01zarSnI15i9jU0QL07aw/edit)

Recommendation:
- Create an additional local term in our [local QA vocabulary definition](https://github.com/emory-libraries/dlp-curate/blob/7e778b1e381d6f9eb140179809f6dbdec316a605/config/authorities/rights_statements.yml) for our _rights_status_ field:  _Copyright Varies by Work_
- If this value is assigned to a work, do not provide a hyperlink in Lux to the new local term value, because it will not resolve to a live page

**Comment by eporter23** [comment]
@nikdragovic updating this ticket based on SCO's recommendation from 6/30/20.

**Comment by nikdragovic** [comment]
@wulman16 Here is a screenshot of what I see from the Add a New Work page in curate-test. Should I see the "Copyright Varies by Work" designation in this list?

![Screen Shot 2020-07-09 at 2.24.28 PM.png](https://images.zenhubusercontent.com/5ba135cf945aef49576d50f2/f86f04b1-01a3-40c8-93f7-30eabde62ab0)

**Comment by wulman16** [comment]
@nikdragovic Yes, from my understanding, it's a field that's available to all new works by default, but is only intended to be used for parent works.

**Comment by nikdragovic** [comment]
@wulman16 Thanks. I tried editing an existing parent work and couldn't get the option to show up in the list. I created a new work and Kathryn helped me attach a child work; still can't get it to show up:

https://curate-test.library.emory.edu/concern/curate_generic_works/689d51c5m4-cor

Hopefully I'm looking in the right place. I don't see any changes in the QA vocab definition that Emily linked to in the ticket description, but I'm not sure if I should be.
